lib/segment/Cargo.toml

```diff
@@ -56,7 +56,7 @@ tar = "0.4.40"
 fs_extra = "1.3.0"
 semver = "1.0.18"
 tinyvec = { version = "1.6.0", features = ["alloc"] }
-quantization = { git = "https://github.com/qdrant/quantization.git" }
+quantization = { git = "https://github.com/qdrant/quantization.git", rev = "ff306d0d990d7286ee1fa3a603daa7efd343ad0c"}
```

Comment:
can we move these version changes to our own monorepo? you should be able to use `[patch]` in our root `Cargo.toml`.